patches/chromium/fix_patch_out_permissions_checks_in_exclusive_access.patch

```diff
@@ -33,19 +90,36 @@ index 8d6f8aedab475c1a553949bfcba3753ebed87778..e379e4995b0812be5970cf9741a00e4f
  #endif
  
    toggled_into_fullscreen_ = true;
-@@ -403,6 +399,7 @@ void FullscreenController::EnterFullscreenModeInternal(
+@@ -421,6 +427,7 @@ void FullscreenController::EnterFullscreenModeInternal(
        url = extension_caused_fullscreen_;
    }
  
 +#if 0
    if (display_id != display::kInvalidDisplayId) {
      // Check, but do not prompt, for permission to request a specific screen.
      // Sites generally need permission to get the display id in the first place.
-@@ -415,6 +412,7 @@ void FullscreenController::EnterFullscreenModeInternal(
+@@ -434,6 +441,7 @@ void FullscreenController::EnterFullscreenModeInternal(
        display_id = display::kInvalidDisplayId;
      }
    }
 +#endif
  
    if (option == BROWSER)
      base::RecordAction(base::UserMetricsAction("ToggleFullscreen"));
+diff --git a/chrome/browser/ui/exclusive_access/fullscreen_controller.h b/chrome/browser/ui/exclusive_access/fullscreen_controller.h
+index 7bd40f52ef5f6b04a7ea114ec4d18c8a98ec6d42..fb04fed5cc1e2e255c9e67c180fababe1fbb3fe0 100644
+--- a/chrome/browser/ui/exclusive_access/fullscreen_controller.h
++++ b/chrome/browser/ui/exclusive_access/fullscreen_controller.h
+@@ -222,10 +222,12 @@ class FullscreenController : public ExclusiveAccessControllerBase {
+   // Used in testing to set the state to tab fullscreen.
+   bool is_tab_fullscreen_for_testing_ = false;
+ 
++#if 0
+   // Tracks related popups that lost activation or were shown without activation
+   // during content fullscreen sessions. This also activates the popups when
+   // fullscreen exits, to prevent sites from creating persisent popunders.
+   std::unique_ptr<PopunderPreventer> popunder_preventer_;
```

Comment:
We should consider adding a dummy implementation for this instead of commenting it all out and bloating the patch

Reply:
It looks like upstream is still churning this code.

I don't have any opinion on `#if 0` vs mock, but considering a mock would make more sense after upstream has stopped churning as much.

Reply:
I think the patch is necessary because we are building chrome/browser/ui/exclusive_access/fullscreen_controller.cc (added via #31572).